File: SpringBoot/shop-founders-modules/shop-founders-core/src/main/java/hf/shopfounders/model/LoginResponse.java
package hf.shopfounders.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class LoginResponse {

    @JsonProperty("username")
    private String username;
    @JsonProperty("password")
    private String password;
    @JsonProperty("role")
    private String role;
    @JsonProperty("loginStatus")
    private LoginStatus loginStatus;
    @JsonProperty("token")
    private String token;

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }

    public LoginStatus getLoginStatus() {
        return this.loginStatus;
    }

    public void setLoginStatus(LoginStatus loginStatus) {
        this.loginStatus = loginStatus;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    @Override
    public String toString() {
        return String.format(
                "LoginResponse[username='%s', password='%s',loginStatus='%s', token='%s']",
                username, password, loginStatus, token);
    }
}

File: AngularBoot/src/app/pages/_models/auth/loginResponse.ts
import {LoginStatus} from './loginStatus';

export class LoginResponse {
    username: string;
    password: string;
    role: string;
    loginstatus: LoginStatus;

    setUsername(username: string) {
        this.username = username;
    }

    getUserName(): string {
        return this.username;
    }

    setPassword(username: string) {
        this.password = username;
    }

    getPassword(): string {
        return this.password;
    }

    setUserRole(role: string) {
        this.role = role;
    }

    getUserRole(): string {
        return this.role;
    }

    setLoginstatus(loginstatus: LoginStatus) {
        this.loginstatus = loginstatus;
    }

    getLoginstatus(): LoginStatus {
        return this.loginstatus;
    }

}
